### DISCUSSION TITLE
Is there a way to use factories.createMany with a db seed?

### DISCUSSION THREAD

**Discussion by codepotato** [adonisjs/core#1297] [Help]
I'm very very new to Adonis but I'm coming from a 5 year background of Laravel so understand the concepts that are being used in Adonis nicely however I've fallen at the first hurdle; seeding db with factory created models! 

To be clear, I've setup mysql and can confirm the connections are working as I have migrated (how damn easy was that!) but I think I'm having a derp moment when attempting to create factory powered models. 

My code at the moment is: 

```export default class UserSeeder extends BaseSeeder {
  public async run () {
    await User.createMany(
      await UserFactory.createMany(10)
    )
  }
}
```

Any ideas why, when I run db:seed, it's attempting to put an empty object as one of the model parameters? 

Just to add, I'm using Adonis v5

**Comment by codepotato** [comment]
A little more debugging here has narrowed it down to my userFactory. 

```
export const UserFactory = Factory
  .define(User, ({ faker}) => {
    return {
      name: faker.name,
      email: faker.internet.email(),
      company_name: faker.company.companyName(),
    }
  })
  .build()

```

When I remove faker.name and replace it with a sting, it works.

**Reply by fnoquiq** [reply]
`faker.name` is a function

`faker.name()` instead of `faker.name`

**Reply by codepotato** [reply]
That's strange, I tried that and it said faker.name() is not a function. 🤷‍♂️

**Reply by fnoquiq** [reply]
Sorry, that's the correct way
`name: faker.name.findName()`


You can find examples [here](https://rawgit.com/Marak/faker.js/master/examples/browser/index.html#name)

**Comment by thetutlage** [comment]
I don't understand. Why you are feeding the output of your factory to the model?

Simply running UserFactory.createMany() will perform the inserts
